setup.py

```diff
@@ -347,10 +344,10 @@ def get_config_var(name):
                                                      remove='lib/cartopy/')) +\
                              list(file_walk_relative('lib/cartopy/data/shapefiles/gshhs',
                                                      remove='lib/cartopy/')) +\
-                             ['io/srtm.json']
+                             ['io/srtm.json'],
+                  '': ['requirements/*.txt'],
```

Comment:
Hmm, yes, but was that not fixed by the `MANIFEST.in`? Or is this necessary too?